cirq-core/cirq/ops/matrix_mixture.py

```diff
@@ -0,0 +1,114 @@
+from typing import Any, Dict, Iterable, Tuple, Union
+import numpy as np
+
+from cirq import linalg, protocols, value
+from cirq._compat import proper_repr
+from cirq.ops import raw_types
+
+
+class MatrixMixture(raw_types.Gate):
```

Comment:
The class represents a channel whose Kraus operators are scalar multiples of unitary matrices. These objects are called "mixed-unitary" or "random unitary" channels in the literature (see e.g. [here](https://quantumcomputing.stackexchange.com/questions/18190/determining-whether-there-exists-an-equivalent-set-of-unitary-kraus-operators)). It would make the class easier to find and understand if it was renamed to `MixedUnitaryChannel`.

Reply:
Done.